SPECS-EXTENDED/hfsplus-tools/hfsplus-tools.spec

```diff
@@ -2,7 +2,7 @@ Vendor:         Microsoft Corporation
 Distribution:   Mariner
 Name:           hfsplus-tools
 Version:        540.1.linux3
```

Comment:
is "linux3" the actual version number?

Reply:
Yep, this is part of the version as shipped by upstream.